Chain Onboarding Doc (#548)
WalletConnect/walletconnect-docs: docs/advanced/chain-onboarding/overview.md

# New Chain Onboarding

Web3 is a multi-chain supported protocol. By using the [CAIP-25 standard](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-25.md), WalletConnect aims to provide a standardized process for onboarding new chains into our ecosystem.

Our Sign and Auth APIs are multi-chain out of the box, along with `@web3modal/standalone`. For those chains wishing to add chain support to `@web3modal/react` or `@web3modal/html`, please see the instructions below.

## Adding a Chain into the Explorer

> If you don't see your chain listed in this list, then you will need to create an issue in GitHub to to get the process started.

Reviewer: Guessing `this list` is a de facto placeholder for a link to be added? Otherwise we should describe where this list is currently visible (i.e. Cloud submission form) until it's consumable via public endpoint.

> You can do so by clicking [here](https://github.com/WalletConnect/walletconnect-monorepo/issues/new?assignees=&labels=type%3A+new+chain+request&template=new_chain_to_explorer.md&title=). Once your chain is added to this list, wallets will be able to register via [Cloud](https://cloud.walletconnect.com) so that users can select your wallet from the options.

Reviewer: ```suggestion
You can do so by clicking [here](https://github.com/WalletConnect/walletconnect-monorepo/issues/new?assignees=&labels=type%3A+new+chain+request&template=new_chain_to_explorer.md&title=). Once your chain is added to this list, wallets & dapps will be able to indicate support for your chain via WalletConnect's [Cloud](https://cloud.walletconnect.com).
```
